app/renderer/lib/appium-client.js

```diff
@@ -229,15 +244,45 @@ export default class AppiumClient {
       await this.driver.switchContext(NATIVE_APP);
     }
 
-    ({platformName, statBarHeight} = await this.driver.getSession());
+    ({platformName, pixelRatio, statBarHeight, viewportRect} = await this.driver.getSession());
+    const isAndroid = _.toLower(platformName) === 'android';
 
     try {
       contexts = await this.driver.executeScript('mobile:getContexts', []);
+      contexts = isAndroid ? this.parseAndroidContexts(contexts) : contexts;
     } catch (e) {
       contextsError = e;
     }
 
+
     if (currentContext !== NATIVE_APP) {
+      try {
+        // Get the webview offset
+        if (viewportRect) {
+          // The viewport rectangles are based on the screen density,
+          // iOS needs CSS pixels
+          webViewPosition = {
+            x: isAndroid ? viewportRect.left : Math.round(viewportRect.left / pixelRatio),
+            y: isAndroid ? viewportRect.top : Math.round(viewportRect.top / pixelRatio),
+          };
+        } else {
+          // Fallback
+        const el = await this.driver.findElement(
+          isAndroid ? 'xpath' : '-ios class chain',
+          isAndroid ? '//android.webkit.WebView' : '**/XCUIElementTypeWebView'
+        );
+          if ('elementId' in el) {
+            // Strange is that this command is failing, isn't it exposed by Web2Driver
```

Comment:
@jlipps 

Do you know why this command is not working, is this an issue with Web2Driver?

Reply:
Thanks, totally missed that, my bad, fixed it now